jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/JarCommandTest.java

```diff
@@ -315,4 +336,45 @@ public void testJar_baseImageSpecified()
     String output = new Command("docker", "run", "--rm", "cli-gcr-base").run();
     assertThat(output).isEqualTo("Hello World");
   }
+
+  public static void createJarFile(String name, String className, String classPath, String mainClass) throws IOException, URISyntaxException {
+    Path javaFilePath = Paths.get(Resources.getResource("jarTest/standard/" + className + ".java").toURI());
+    Path workingDir = Paths.get(Resources.getResource("jarTest/standard/").toURI());
+    System.out.println("Working dir: " + workingDir);
```

Comment:
How about we log this message?

Reply:
This was for debugging. I will just remove it.